app/src/main/java/net/bible/android/view/activity/navigation/GridChoosePassageBook.kt

```diff
@@ -16,6 +16,7 @@
  *
  */
 
+// TODO: Not saving the menu option. Don't know how to set the other menu options when one option is changed.
```

Comment:
- [ ] TODO

Reply:
You just save as normal, and then invalidate options (see my commits for how to do it). I think it is done correctly now, right?